### PR TITLE
autodoc: Add / as an alternative search key

### PR DESCRIPTION
**Pull request by cryptocode**
Per Discord discussion. Motivation: `/` is now the de facto standard for initating search on webpages.

### CODE REVIEW

**Comment by r00ster91** [comment]
I agree this one is much better. But why not make this the only key? I think we can just remove support for <kbd>S</kbd>?
My initial worry was that in for example Firefox <kbd>/</kbd> opens up this Quick Find (which is like Ctrl+F) but `preventDefault` which is already there should take care of this.
On GitHub as well <kbd>/</kbd> does not open Firefox' Quick Find but GitHub's search.

File: lib/docs/index.html [hunk 1]
```diff
@@ -758,7 +758,7 @@ <h2><span>Zig Version</span></h2>
         <div class="wrap">
           <section class="docs" style="padding-top: 1.5rem; padding-bottom:0;">
           <div style="position: relative">
-            <span id="searchPlaceholder"><kbd>s</kbd> to search, <kbd>?</kbd> for more options</span>
+            <span id="searchPlaceholder"><kbd>/</kbd> or <kbd>s</kbd> to search, <kbd>?</kbd> for more options</span>
```
Comment (r00ster91) [inline lib/docs/index.html:761]: You also have to update a line
```
  879           <dl><dt><kbd>s</kbd></dt><dd>Focus the search field</dd></dl>
```
way down and change "s" to "/" (again, I recommend just removing support for <kbd>s</kbd>).

**Comment by cryptocode** [comment]
@r00ster91 1) Loris wanted to keep 's' in addition to '/' for now, and 2) Andrew also mentioned the potential Firefox issue, but acknowledged that / is the defacto search key by now (but as you say, shouldn't be a problem)

File: lib/docs/index.html (hunk 1 above)
Comment (cryptocode) [reply lib/docs/index.html:761]: Done

**Pull request merged**